src/libs/antares/study/area/list.cpp

```diff
@@ -45,6 +46,178 @@ namespace Antares::Data
 {
 namespace // anonymous
 {
+
+enum tristate : uint8_t
+{
+    False = 0,
+    True = 1,
+    Undefined = 2
+};
+
+void readReservesAreaParameters(Antares::Data::Area& area, const IniFile::Section& section)
+{
+    for (auto* p = section.firstProperty; p; p = p->next)
+    {
+        std::string key = p->key;
+        std::ranges::transform(key, key.begin(), [](unsigned char c) { return std::tolower(c); });
+
+        if (key == "energy-activation-ratio-up")
+        {
+            if (!p->value.to<double>(
+                  area.allCapacityReservations.value()
+                    .maxGlobalEnergyActivationRatio[(int)reserve::Direction::UP]))
+            {
+                logs.warning() << area.name
+                               << ": invalid maximum energy activation ratio for UP reserves";
+            }
+        }
+        else if (key == "energy-activation-ratio-down")
+        {
+            if (!p->value.to<double>(
+                  area.allCapacityReservations.value()
+                    .maxGlobalEnergyActivationRatio[(int)reserve::Direction::DOWN]))
+            {
+                logs.warning() << area.name
+                               << ": invalid maximum energy activation ratio for "
+                                  "DOWN reserves";
+            }
+        }
+        else if (key == "reference-activation-duration-up")
+        {
+            if (!p->value.to<int>(
+                  area.allCapacityReservations.value()
+                    .referenceGlobalActivationDuration[(int)reserve::Direction::UP]))
+            {
+                logs.warning() << area.name
+                               << ": invalid reference energy activation duration "
+                                  "for UP reserves";
+            }
+        }
+        else if (key == "reference-activation-duration-down")
+        {
+            if (!p->value.to<int>(
+                  area.allCapacityReservations.value()
+                    .referenceGlobalActivationDuration[(int)reserve::Direction::DOWN]))
+            {
+                logs.warning() << area.name
+                               << ": invalid reference energy activation duration "
+                                  "for DOWN reserves";
+            }
+        }
+    }
+}
+
+void readReserveParameters(fs::path& folderInput,
+                           Antares::Data::Area& area,
+                           const IniFile::Section& section,
+                           bool* ret)
```

Comment:
to follow the legacy code, you need to return a boolean
```suggestion
                           bool ret)
```